iFixit/react-commerce
# Troubleshooting Collection Page: Start using the API

QA
---
Similar to https://github.com/iFixit/ifixit/pull/49739
* Ensure Breadcrumbs match the API
* Ensure a 404 is returned in the same ways
Connects: https://github.com/iFixit/ifixit/issues/49588
